src/main/java/uk/gov/cshr/civilservant/controller/CivilServantController.java

```diff
@@ -61,12 +82,61 @@ public CivilServantController(CivilServantRepository civilServantRepository,
         return getResourceResponseEntity(optionalCivilServant);
     }
 
+    @GetMapping("/manager")
+    @PreAuthorize("isAuthenticated()")
+    public ResponseEntity<Resource<CivilServantResource>> check(@RequestParam(value = "email") String email) throws NotificationClientException {
```

Comment:
SOLID - Controller does lots of things, either extract to private succinct methods or delate to a service.

Reply:
Moved to LineManagerService